nanovdb/nanovdb/tools/GridBuilder.h

```diff
@@ -1158,7 +1158,6 @@ struct LeafNode
         ValueIterator& operator=(const ValueIterator&) = default;
         ValueType operator*() const { NANOVDB_ASSERT(*this); return mParent->mValues[mPos];}
         Coord getCoord() const { NANOVDB_ASSERT(*this); return mParent->offsetToGlobalCoord(mPos);}
-        bool isActive() const { NANOVDB_ASSERT(*this); return mParent->isActive(mPos);}
```

Comment:
can you change the implementation to this:

`bool isActive() const { NANOVDB_ASSERT(*this); return mParent->mValueMask.isOn(mPos);}`

Reply:
if that also fails then I agree that we can simply remove this method

Reply:
Ah yeah, that works. I updated the PR with the change.